src/components/Speciality/Training.js

```diff
@@ -5,14 +5,14 @@ import { Padding } from 'styled-components-spacing'
 import GreyBackground from '../Common/GreyBackground'
 import StyledLink from '../Common/StyledLink'
 
-const TrainingStage = ({ title, body, icon }) => (
+const TrainingStage = ({ title, body, iconTitle, iconUrl }) => (
   <Col width={[1, 1, 1, 1, 4 / 12]}>
     <Padding top={4} bottom={4}>
       <Padding bottom={1.5}>
```

Comment:
Please remove all `Padding` from this file

Reply:
No more Padding here!